### PR TITLE
[Kotlin Migration] ClosableDrawerLayout

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] ClosableDrawerLayout

### CODE REVIEW

File: AnkiDroid/kotlinMigration.gradle [hunk 1]
```diff
@@ -43,7 +43,7 @@ permission notice:
 // Example of class name: "/com/ichi2/anki/UIUtils.kt"
 // Ensure that it starts with '/' (slash)
 def source = Source.MAIN
-def className = ""
+def className = "/androidx/drawerlayout/widget/ClosableDrawerLayout.kt"
```
Comment (david-allison-1) [inline AnkiDroid/kotlinMigration.gradle]: Abnormal. Has this commit been tested? I've only tested with `com`

**Review by david-allison-1 — COMMENTED** [review]
One question regarding testing. Code looks good

File: AnkiDroid/kotlinMigration.gradle (hunk 1 above)
Comment (codingtosh) [reply AnkiDroid/kotlinMigration.gradle]: Well, this commit was building without any errors. Should I run tests locally or wait for the CI to do that?

Comment (david-allison-1) [reply AnkiDroid/kotlinMigration.gradle]: Na, if the commit builds we're good to go

**Review by david-allison-1 — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]

**Pull request merged**